### PR TITLE
add css handles

### PR DESCRIPTION
**Pull request by jgfidelis**
#### What problem is this solving?

Add more css classes in shipping simulator

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/components/ShippingSimulator/index.js [hunk 1]
```diff
@@ -88,25 +87,28 @@ const ShippingSimulator = ({
...
-          className={styles.shippingCTA}
-          disabled={!isValid}
-          size="small"
-          type="submit"
-          block
-          isLoading={loading}
-        >
-          {intl.formatMessage({ id: 'store/shipping.label' })}
-        </Button>
+        <div className={styles.shippingButton}>
```
Comment (lbebber) [inline react/components/ShippingSimulator/index.js]:
```suggestion
        <div className={styles.shippingButtonContainer}>
```

File: react/components/ShippingSimulator/shippingSimulator.css [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+}
+.shippingTableCell {
+}
+.shippingTableLabel {
+}
+.shippingTableRadioBtn {
+}
+.shippingButtonText {
+}
+.shippingButton {
```
Comment (lbebber) [inline react/components/ShippingSimulator/shippingSimulator.css]:
```suggestion
.shippingButtonContainer {
```

File: docs/ShippingSimulator.md [hunk 3]
```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
...
+| Prop name | Type      | Description                   | Default value |
+| --------- | --------- | ----------------------------- | ------------- |
+| `skuId`   | `String!` | Id of the current product SKU | -             |
+| `seller`  | `String!` | Id of the product seller      | -             |
 
 ### Styles API
 
 You should follow the Styles API instruction in the main [README](/README.md#styles-api).
 
 #### CSS Namespaces
```
Comment (iaronaraujo) [inline docs/ShippingSimulator.md]:
```suggestion
#### Customization
```

File: docs/ShippingSimulator.md [hunk 4]
```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
...
-| `shippingZipcodeLabel` | The shipping label | [index](/react/components/ShippingSimulator/index.js) |
-| `shippingCTA` | The calculate shipping button | [index](/react/components/ShippingSimulator/index.js) |
-| `shippingNoMessage` | The message when the store does not have a shipping method | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js) |
-| `shippingTable` | The table for the shipping methods | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js) |
-| `shippingTableCell` | The cell of a table | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
-| `shippingTableLabel` | The label describing a shipping method | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
-| `shippingTableRadioBtn` | The radio button to select a shipping method | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
\ No newline at end of file
+| Class name                   | Description                                                | Component Source                                                                       |
```
Comment (iaronaraujo) [inline docs/ShippingSimulator.md]:
```suggestion
| CSS Handles                  | Description                                                | Component Source                                                                       |
```

File: docs/ShippingSimulator.md [hunk 5]
```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
...
+| `shippingZipcodeLabel`       | The shipping label                                         | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingNoMessage`          | The message when the store does not have a shipping method | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js)       |
+| `shippingTable`              | The table for the shipping methods                         | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js)       |
+| `shippingTableCell`          | The cell of a table                                        | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
+| `shippingTableLabel`         | The label describing a shipping method                     | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
+| `shippingTableRadioBtn`      | The radio button to select a shipping method               | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
```
Comment (iaronaraujo) [inline docs/ShippingSimulator.md]: Add this text below the handles: 
"In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization). "

File: react/components/ShippingSimulator/index.js [hunk 6]
```diff
@@ -88,25 +90,27 @@ const ShippingSimulator = ({
...
-          className={styles.shippingCTA}
-          disabled={!isValid}
-          size="small"
-          type="submit"
-          block
-          isLoading={loading}
-        >
-          {intl.formatMessage({ id: 'store/shipping.label' })}
-        </Button>
+        <div className={handles.shippingButtonContainer}>
```
Comment (iaronaraujo) [inline react/components/ShippingSimulator/index.js]: This additional div can cause some problems

File: react/components/ShippingSimulator/index.js [hunk 7]
```diff
@@ -88,25 +90,27 @@ const ShippingSimulator = ({
...
+            onClick={handleClick}
+            disabled={!isValid}
+            size="small"
+            type="submit"
+            block
+            isLoading={loading}
+          >
+            <div className={handles.shippingButtonText}>
+              <FormattedMessage id="store/shipping.label" />
+            </div>
```
Comment (iaronaraujo) [inline react/components/ShippingSimulator/index.js]:
```
<FormattedMessage id="store/shipping.label">
 {message => <span className={handles.shippingButtonText}>{message}</span>}
</FormattedMessage>
```

File: react/components/ShippingSimulator/components/ShippingTable.js [hunk 8]
```diff
@@ -1,12 +1,11 @@
...
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
+import { withCssHandles } from 'vtex.css-handles'
 
 import ShippingTableRow from './ShippingTableRow'
 
-import styles from '../shippingSimulator.css'
-
-export default class ShippingTable extends Component {
+class ShippingTable extends Component {
```
Comment (klzns) [inline react/components/ShippingSimulator/components/ShippingTable.js]: (Could have changed to function component)

File: react/components/ShippingSimulator/index.js (hunk 7 above)
Comment (jgfidelis) [reply react/components/ShippingSimulator/index.js]: por que span? não tem diferença nesse caso

File: docs/ShippingSimulator.md (hunk 3 above)
Comment (klzns) [inline docs/ShippingSimulator.md]: Change to "Customization"

File: docs/ShippingSimulator.md [hunk 9]
```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
...
+| `skuId`   | `String!` | Id of the current product SKU | -             |
+| `seller`  | `String!` | Id of the product seller      | -             |
 
 ### Styles API
 
 You should follow the Styles API instruction in the main [README](/README.md#styles-api).
 
 #### CSS Namespaces
+
 Below, we describe the namespace that are defined in the `ShippingSimulator`.
```
Comment (klzns) [inline docs/ShippingSimulator.md]: Remove line

File: docs/ShippingSimulator.md (hunk 4 above)
Comment (klzns) [inline docs/ShippingSimulator.md]:
```suggestion
| CSS Handle                   | Description                                                |
```

Remove component source

Comment (klzns) [reply docs/ShippingSimulator.md]: Check https://github.com/vtex-apps/breadcrumb for reference

File: react/components/ShippingSimulator/components/ShippingTable.js (hunk 8 above)
Comment (jgfidelis) [reply react/components/ShippingSimulator/components/ShippingTable.js]: not the purpose of this PR

File: docs/ShippingSimulator.md [hunk 10]
```diff
@@ -40,11 +40,9 @@ You can use it in your code like a React component with the jsx tag: `<ShippingS
 
 You should follow the Styles API instruction in the main [README](/README.md#styles-api).
 
-#### CSS Namespaces
+#### Customization
 
-Below, we describe the namespace that are defined in the `ShippingSimulator`.
-
-| Class name                   | Description                                                | Component Source                                                                       |
+| CSS Handle                   | Description                                                | Component Source                                                                       |
```
Comment (klzns) [inline docs/ShippingSimulator.md]: Remove the last column

Comment (iaronaraujo) [reply docs/ShippingSimulator.md]: You could also only keep the `CSS Handle` column

File: react/components/ShippingSimulator/index.js (hunk 7 above)
Comment (iaronaraujo) [reply react/components/ShippingSimulator/index.js]: Tem sim. O formattedMessage por natureza ja gera um span, então é melhor vc colocar className no span do jeito que eu coloquei aqui do que criar uma div que vai envolver o span

File: react/components/ShippingSimulator/index.js (hunk 6 above)
Comment (jgfidelis) [reply react/components/ShippingSimulator/index.js]: vi nas lojas e não vai afetar ngm

File: react/components/ShippingSimulator/index.js (hunk 7 above)
Comment (iaronaraujo) [inline react/components/ShippingSimulator/index.js]: The change I've requested is still pending

File: react/components/ShippingSimulator/index.js (hunk 6 above)
Comment (iaronaraujo) [inline react/components/ShippingSimulator/index.js]: Didn't this cause a problem in a store?

File: CHANGELOG.md [hunk 11]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- More CSS Handles for `ShippingSimulator`.
+
+	### Changed
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
### Changed
```

File: docs/ShippingSimulator.md [hunk 12]
```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
...
-You can use it in your code like a React component with the jsx tag: `<ShippingSimulator />`. 
+You can use it in your code like a React component with the jsx tag: `<ShippingSimulator />`.
 
 ```jsx
-<ShippingSimulator
-  skuId="3"
-  seller="1"
-/>
+<ShippingSimulator skuId="3" seller="1" />
 ```
```
Comment (victorhmp) [inline docs/ShippingSimulator.md]: Should we keep these instructions?

File: react/__tests__/components/__snapshots__/ShippingSimulator.test.js.snap [hunk 13]
```diff
@@ -16,15 +16,24 @@ exports[`<ShippingSimulator /> component should match snapshot with skuId and se
...
+      >
+        <div
+          class="shippingButtonText"
+        >
+          <span>
+            Calculate shipping
+          </span>
+        </div>
+      </button>
+    </div>
```
Comment (victorhmp) [inline react/__tests__/components/__snapshots__/ShippingSimulator.test.js.snap]: Wouldn't this cause the IO Compras bot to fail?

**Comment by lbebber** [comment]
This PR is by now a little too old and stale, so closing.

If needed, a new PR based on this branch can be opened (or rebase and reopen this PR)

**Pull request closed without merging**